test/gtest/Makefile.am

```diff
@@ -19,6 +19,8 @@ VALGRIND_EXTRA_ARGS      ?=
 
 export UCX_HANDLE_ERRORS
 export UCX_LOG_LEVEL
+export UCX_MEM_LOG_LEVEL
```

Comment:
how is it related?

Reply:
not related, small fix that was found it during debug process